docs/contributing.md

```diff
@@ -1,5 +1,6 @@
 # Contributing
-Contributions are welcome. We have a few requests for new contributors:
+Contributions are welcome.
+We have a few asks of contributors, whether they be for core, webhost, or new game contributions:
```

Comment:
I think "asks of" is quite formal, to the point that it's almost awkward. "requests for" or even "requirements for" would both be better imho